charts/astronomer/templates/_helpers.yaml

```diff
@@ -304,3 +306,12 @@ s3:
 {{- end }}
 {{- end }}
 {{- end }}
+
+{{- define "houston.nats.servers"}}
+{{- $ns := dict "servers" (list) -}}
+{{- $name := printf "%s-nats" .Release.Name -}}
+{{- range $i, $e := until (.Values.global.natsReplicas | int) -}}
+{{- $noop := printf "\"%s-%d.%s.%s.svc:4222\"" $name $i $name $.Release.Namespace | append $ns.servers | set $ns "servers" -}}
+{{- end }}
+{{- printf "'[%s]'" (join "," $ns.servers) -}}
```

Comment:
Because this is formatted as: 
```
'["astronomer-nats-0.astronomer-nats.astronomer.svc:4222","astronomer-nats-1.astronomer-nats.astronomer.svc:4222","astronomer-nats-2.astronomer-nats.astronomer.svc:4222"]'
```

It will require the application code to parse this JSON object or is this coming in as the same format as the default yaml?  

```
servers: ["nats://localhost:4222"]
```

Reply:
we need json string here more details in https://github.com/astronomer/houston-api/pull/436/files#diff-b3e5513342454bc95092ef963d64b431R16